### PR TITLE
Snippets for haml templates

### PR DESCRIPTION
**Pull request by arandilopez**
This only covers the views.cson... wip for views_embbed.cson

### CODE REVIEW

**Comment by arandilopez** [comment]
now covers forms and inputs

**Comment by arandilopez** [comment]
embedded lines snippets

File: snippets/forms_and_inputs_haml.cson [hunk 1]
```diff
@@ -0,0 +1,126 @@
+'.text.haml':
```
Comment (arandilopez) [inline snippets/forms_and_inputs_haml.cson]: this is a new file with your forms and inputs (erb) snippets but in haml sintax

File: snippets/views-haml.cson [hunk 2]
```diff
@@ -0,0 +1,184 @@
+'.text.haml':
```
Comment (arandilopez) [inline snippets/views-haml.cson]: this new file is your views snippets (erb) but in haml sintax...

Note: snippets don't have `<%  %>` tags

File: snippets/views_embedded_ruby.cson [hunk 3]
```diff
@@ -27,6 +27,34 @@
       <% end
     """
 
+'.text.haml > .line.ruby.haml > .source.ruby':
```
Comment (arandilopez) [inline snippets/views_embedded_ruby.cson]: I divide these snippet in 3 selectors group: Only erb, only haml, both erb and haml

File: snippets/views_embedded_ruby.cson [hunk 4]
```diff
@@ -27,6 +27,34 @@
...
+    """
+
+  'haml embedded block with parm':
+    'prefix': 'dop'
+    'body': """
+      do |${1:member}|
+        = ${1:member}$2
+    """
+
+'.text.html.erb > .embedded.line.erb > .source.ruby,
```
Comment (arandilopez) [inline snippets/views_embedded_ruby.cson]: Selector for both erb and haml, these snippets don't use `<% %>` tags so You can use with haml

File: snippets/views_embedded_ruby.cson [hunk 5]
```diff
@@ -27,6 +27,34 @@
       <% end
     """
 
+'.text.haml > .line.ruby.haml > .source.ruby':
+  'haml embedded each':
+    'prefix': '.each'
+    'body': """
+      .each do |${1:member}|
```
Comment (arandilopez) [inline snippets/views_embedded_ruby.cson]: this snippet is rewritten for haml syntax

**Comment by joseramonc** [comment]
Thanks for your help!

:shipit: :shipit: :shipit:

**Pull request merged**